extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -433,6 +433,34 @@ public function getCuratedContentQuality() {
 		}
 	}
 
+	public function getData( ) {
+		global $wgWikiaCuratedContent;
+		$data = [];
+
+		if ( !empty( $wgWikiaCuratedContent ) && is_array( $wgWikiaCuratedContent )  ) {
+			foreach ( $wgWikiaCuratedContent as $section ) {
+				// sections
+				if ( !empty( $section['title'] ) && empty( $section['featured'] ) ) {
+					list( $_image_id, $url ) = CuratedContentSpecialController::findImageIfNotSet( $section['image_id'] );
```

Comment:
`list( , $url ) = CuratedContentSpecialController::findImageIfNotSet( $section['image_id'] );`
you can skip values